src/test/unit/controller/ApplicationControllerTests.ts

```diff
@@ -0,0 +1,233 @@
+import { expect } from 'chai';
+import sinon from 'sinon';
+import express from "express";
+import * as ApplicationContoller from "../../../main/controllers/ApplicationController";
+import * as ApplicationService from "../../../main/services/ApplicationService";
+import * as AwsUtil from "../../../main/Utils/AwsUtil"
+import jwt from 'jsonwebtoken';
+import { allowRoles } from "../../../main/middleware/AuthMiddleware";
+import { UserRole } from "../../../main/models/JwtToken";
+
+declare module 'express-session' {
+  interface SessionData {
+    token: string;
+  }
+}
+
+interface testReq1 {
+  session: {
+    token: string;
+  };
+  params: {
+    id: number;
+  };
+  file: File | null;
+}
```

Comment:
is this needed?

Reply:
yes, because of one of the tests ("should return error when no file is uploaded" test) needs the file req to be null. without the above code i get the following error: Object literal's property 'file' implicitly has an 'any' type.